### DISCUSSION TITLE
Github Sponsor Payout never issues (first-time new user of Github Sponsor)

### DISCUSSION THREAD

**Discussion by jhcarl0814** [community/community#67693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Topic Area

Bug

### Body

I got a one-time sponsorship on 2023-08-17 and a monthly sponsorship starting from 2023-02. But the payment is stuck:

In Github Sponsor dashboard it says:
```
ESTIMATED NEXT PAYOUT [month] 22
```
and whenever that day comes, it becomes same day of the next month:
```
ESTIMATED NEXT PAYOUT [next month] 22
```

In Stripe page it says:
```
Payout expected on 22 [month]
```
and whenever that day comes, it becomes same day of the next month:
```
Payout expected on 22 [next month]
```
----------
## Is there a place in Github Sponsor dashboard or Stripe page that tells me why it never issues payout, and what actions should I take to fix this?

**Comment by DHayes10** [comment]
Hi @jhcarl0814, 

We are sorry to hear about you not receiving your monthly payouts. I have raised this issue with our team who will take a deeper look and get this resolved for you as soon as possible. I will follow up within 2-3 business days with an update. Thanks for getting in touch!

Best, 
GitHub Sponsors

**Comment by DHayes10** [comment]
Hi @jhcarl0814 it appears Stripe is requiring additional information in order to receive payouts for your Sponsors account. Please contact https://support.stripe.com/contact for more assistance. 

Thank you, 
GitHub Support

**Comment by jhcarl0814** [comment]
Hi @DHayes10 ,

After Stripe Support told me that I'm "all good to go in regards to taking in new charges or paying out funds", the payout didn't pay and the payout date changed to next month again. Then they suggest me to contact Github, because the one who can schedule the payouts is Github (I should be guided and assisted by Github related to my payout schedule):

![image](https://github.com/community/community/assets/5171262/0fc80a5e-3c4d-4a8a-b83b-6063d0f3c6b1)

**Reply by jhcarl0814** [reply]
@lsgdefault After the manual payout in https://github.com/orgs/community/discussions/72298 , auto payout (at the end of each month) starts working.

**Reply by GTTBank** [reply]
Yes
